packages/create-component/scaffolders/constants.js

```diff
@@ -0,0 +1,29 @@
+const renames = {
+    _gitignore: '.gitignore',
+    '_package.json': 'package.json',
+    '_README.md': 'README.md',
+    '_settings.json': 'settings.json',
```

Comment:
why this renames needed?

Reply:
.gitignore in a subdirectory makes git ignore files present in the subdirectory which is something we might not want.

Reply:
If not need I shall remove it.

Reply:
not need da remove it

Reply:
https://github.com/kissflow/lcnc-sdk-js/pull/117#discussion_r1820216866 - This thread has full explanation, if not needed I shall refactor it.